src/commands/run.ts

```diff
@@ -106,6 +106,7 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
 	};
 
 	async run() {
+		console.log('running');
```

Comment:
i guess this one was only for debugging and shouldn't be here

Reply:
WOOPS, indeed